### ISSUE TITLE
Allow more than one folder for the game files

### ISSUE THREAD

**Issue by spaceboy1234** [Phalcode/gamevault-backend#329]
I suggest to allow more than one folder that the backend could search for game files. Games are bigger and bigger so it would be useful to distribute the games on more than one hard disk.

Thanks !

**Comment by Alfagun74** [comment]
There's no need. Just mount a subfolder as a docker volume. Like /storage/disk2.

Or if you're not using docker, just use symlinks.

**Comment by spaceboy1234** [comment]
I tried with symbolic links (using [this](https://download.cnet.com/link-shell-extension-64-bit/3000-2248_4-75213087.html)) and putting a symbolic link to another drive as a subfolder of my original /files folder and it doesn't work. It is ignored when scanning the games. Any hints on how to make it work ?

**Comment by Alfagun74** [comment]
do I understand correctly you are not using docker?

**Comment by spaceboy1234** [comment]
Yes I am using Docker but I don't use a Docker volume for the game files, I use a windows path directly instead. Anyway, I checked a little bit and I don't understand how to point to 2 different drives using a Docker volume. Can I do this in the Docker Desktop interface or it's via the compose configuration file ?

**Comment by Alfagun74** [comment]
show me your compose file, I'll show you how to edit it.

**Comment by spaceboy1234** [comment]
Hi !

Sorry for the late reply, I was not available.

I saw that you released a new version that supports symlinks ! I updated my
back-end to the latest version and I tried it but it does not seem to
work...

Here is my compose file :
services:
  gamevault-backend:
    image: phalcode/gamevault-backend:latest
    restart: unless-stopped
    environment:
      DB_HOST: db
      DB_USERNAME: gamevault
      DB_PASSWORD: xxxxxx
      METADATA_IGDB_CLIENT_ID: xxxxx
      METADATA_IGDB_CLIENT_SECRET: xxxxx
      SERVER_REGISTRATION_DISABLED: false
      SERVER_ACCOUNT_ACTIVATION_DISABLED: true
    volumes:
      # Mount the folder where your games are
      - s:/GameVault/Games:/files
      # Mount the folder where GameVault should store its media
      - s:/GameVault/Media:/media
    ports:
      - 8080:8080/tcp
  db:
    image: postgres:16
    restart: unless-stopped
    environment:
      POSTGRES_USER: gamevault
      POSTGRES_PASSWORD: xxxxx
      POSTGRES_DB: gamevault

**Comment by Alfagun74** [comment]
If you are using Docker anyway, there is no need to use symbolic links. Just do it like this:
https://gamevau.lt/docs/server-docs/structure#mounting-multiple-game-volumes

**Comment by spaceboy1234** [comment]
Hi !

Thank you for your reply !
The "mnt/" syntax did not work for me but I made it work like this :

volumes:
      # Mount the folder where your games are
      - s:/GameVault/Games:/files
      - l:/GameVault/Games:/files/l

Thanks !

**Comment by Alfagun74** [comment]
oh the mnt paths were just examples to enter your own, you did it fully correct, awesome!
